README.md

```diff
@@ -1,108 +1,45 @@
-
-##### *Branches*
-As you work on a git repo the first branch you are on is usually the default branch. This is often called `master`. If you start working on a section of the website (say the footer styling), it is best practise to create your own branch for that feature. Creating your own branch is like taking a copy of `master` and renaming it. When you commit, they will now be on that new branch only.
-
-In the sections below we will walk through how to do this. For the meantime, just note that you always have one default branch, and can have as many other branches as needed.
-
-
-<a name="version-control" id="version-control"></a>
-## Why Version Control
+1. [Introducing Github Flow](#github-flow)
+2. [Git Basics](#git-basics)
+  + [Getting Started](#getting-started)
+  + [Branching](#branching)
+  + [Making Changes](#changes)
+  + [Merging with Master](#merging)
+  + [Merge Conflicts](#conflicts)
+  + [Changing File Structure](#changing-file-structure)
+
+3. [Git for Collaboration](#git-collaboration)
+  + [Terminology](#terminology)
+  + [The Timeline](#timeline)
+  + [Committing](#committing)
+
+4. [Resources](#resources)
 
-##### Code Base History
-Git can provide you with a complete history of every commit made on a project. Benefits include:
-
-* Being able to see what differences to the file system you have made before you have commited.
-
-* Being able to see what differences exist between commits
-
-* Being able to move between different commits (or places in time). This is especially useful if something became broken while you were working on it, and you need to start again.
-
-In the introduction section of this tutorial, we won't be able to cover all these benefits in practise extensively. However, we will aim to give you all the information you will need to know by the end.
-
-##### Multiple people working on the same files
-Version control makes this possible. If you work on one file, and then I work on the same file at the same time when we want to combine our changes git allows us to keep both versions save that we can compare. This allows us to integrate our changes more swiftly.
+<a name="github-flow" id="github-flow"></a>
+## GitHub Flow
+Version control makes it possible for you and I to work on one file at the same time, because we can keep both our versions saved and then compare them once we have each finished our work. This allows us to integrate our changes more swiftly.
 
-We will practice with this later.
+Most developers use git as their version control system, but different teams use different "workflows". At Founders and Coders, we generally follow something called "GitHub flow", because this flow makes it easy to deploy the latest version of your application very regularly. For a fuller explanation, there is a useful article in the resources section below.
```

Comment:
> Most developers use git as their version control system

Should be a link maybe to here: https://rhodecode.com/insights/version-control-systems-2016